helm/cluster/values.yaml

```diff
@@ -43,11 +43,18 @@ internal:
           group: controlplane.cluster.x-k8s.io
           kind: KubeadmControlPlane
           version: v1beta1
+  kubeadmConfig:
```

Comment:
Some improvements in the JSON schema can make these default values not to be here (so it's less clutter on the eyes when looking at the values), but we can optimize these details later.